### DISCUSSION TITLE
spacy ray maintained?

### DISCUSSION THREAD

**Discussion by python3Berg** [explosion/spaCy#11139] [Help: Installation] [answered]
I see few questions about spacy-ray and none that are recent. It is also is limited to 3.1...Given its prominence in documentation, are there any plans to keep up to date. Are people using ray or have they developed alternatives...Given never ending need for speed, I had hoped to use when migrating from 2.3. Is this something that will become more mainstream?

**Comment by rmitsch** [comment, marked as answer]
Hi @python3Berg! We're definitely planning to update (and maintain) it, but are not sure yet when we'll get around to doing that.
